.rubocop.yml

```diff
@@ -177,14 +177,26 @@ Naming/FileName:
   Enabled: false
   Exclude: []
 
-Layout/FirstParameterIndentation:
-  Description: Checks the indentation of the first parameter in a method call.
+Layout/IndentFirstArgument:
+  Description: 'Checks the indentation of the first argument in a method call.'
   Enabled: true
+  VersionAdded: '0.68'
```

Comment:
this version is different then the version in the `.hound.yml` does that matter?

Reply:
https://github.com/rubocop-hq/rubocop/issues/5978
it's just a meta data that specify which version was this added
I copied it from https://github.com/rubocop-hq/rubocop/blob/64f6ca77af6aa14dc754f9cc5a8fbf32d4fa4f74/config/default.yml#L669
Let me remove it then